Project.toml

```diff
@@ -14,9 +13,10 @@ FillArrays = "0.11"
 julia = "1.5"
 
 [extras]
+BandedMatrices = "aae01518-5342-5314-be14-df237901396f"
```

Comment:
It's a very bad idea to have a package (BandedMatrices) that depends on this package in the tests: as soon as a breaking minor version increment is made the tests cannot possibly pass

Reply:
Right, I hadn't thought about that.